src/server.h

```diff
@@ -2558,7 +2559,7 @@ struct serverCommand {
                                     * still maintained (if applicable) so that
                                     * we can still support the reply format of
                                     * COMMAND INFO and COMMAND GETKEYS */
-    dict *subcommands_dict;        /* A dictionary that holds the subcommands, the key is the subcommand sds name
+    hashset *subcommands_dict;        /* A set that holds the subcommands, the key is the subcommand sds name
```

Comment:
The name suggests that it's a dict. We can call it `subcommands_set` or just `subcommands`.